src/screens/Ticketing/Purchase/utils.ts

```diff
@@ -29,3 +31,27 @@ export const getPurchaseFlow = (
     };
   }
 };
+
+export function getExpireDate(iso: string): string {
+  let date = parseISO(iso);
+  let year = date.getFullYear();
+  let month = date.getMonth();
+  if (month === 0) {
+    month = 12;
+    year--;
+  }
+  return `${month < 10 ? '0' + month : month}/${year.toString().slice(2, 4)}`;
+}
+
+export function getPaymentTypeName(paymentType: PaymentType) {
+  switch (paymentType) {
+    case PaymentType.Visa:
+      return 'Visa';
+    case PaymentType.Mastercard:
+      return 'MasterCard';
+    case PaymentType.Vipps:
```

Comment:
We technically have also BankAxept and American Express. Not something to handle here probably but more FYI